src/runtime_src/xdp/profile/device/device_intf.cpp

```diff
@@ -523,4 +513,89 @@ DeviceIntf::~DeviceIntf()
     }
   }
 
+  bool DeviceIntf::initTs2mm(uint64_t bo_size)
+  {
+    xrt::device* xrtDevice = (xrt::device*)mDeviceHandle;
```

Comment:
No c-style cast in c++ code.   and when casting, please assign to auto, why repeat the time both left and right?

Reply:
fixed